toolbox.json

```diff
@@ -21,6 +21,11 @@
     "fixedKeys": [
       "extends",
       "compilerOptions"
-    ]
+    ],
+    "pathMapping": {
+      "include": [
+        "packages/apps/plugins/*"
```

Comment:
if I understand correctly, this means only the plugins will hot reload? lower level libraries will still require recompilation unless they're added to this list?

Reply:
yea, we can add them progressivelly as long as they are compiled "normally" (without any special plugins)

Reply:
makes sense 👍 just wanted to test my understanding. In the long run lower level libs (e.g. protocols, dxos-ui, etc.) will be coming from another repo so this seems like the right level to set this at. Lower levels we should be test with unit tests/storybooks vs hotreloading composer